kapajira/jira/reporter.py

```diff
@@ -27,41 +27,45 @@ def issue_exists(self, issue_hash: str) -> list:
             issue_hash: hash that will be searched in JIRA description field
 
         Returns:
-            True if there is at least one JIRA issue which is not in status 'Closed'
-            and contains issue_hash, False otherwise
+            first JIRA issue which is not in status 'Closed'
+            and contains issue_hash or None
 
         """
         issues = self._search_for_issues(issue_hash, self.STATUS_CLOSED)
 
-        return True if issues else False
+        if issues:
+            return issues[0]
+        else:
+            return None
+
 
     def create_issue(self, issue):
```

Comment:
Heh I had that and then changed back to make it shorter. Ok I'll rename again :)